src/standalone/Soil/soil_hydrology_parameterizations.jl

```diff
@@ -7,7 +7,107 @@ export volumetric_liquid_fraction,
     impedance_factor,
     viscosity_factor,
     dψdϑ,
-    is_saturated
+    update_albedo!
+
+"""
+    albedo_from_moisture(surface_liquid_fraction::FT, albedo_dry::FT, albedo_wet::FT)
+
+Calculates pointwise albedo for any band as a function of soil effective saturation given
+the dry and wet albedo values for that band.
+"""
+function albedo_from_moisture(
+    surface_liquid_fraction::FT,
+    albedo_dry::FT,
+    albedo_wet::FT,
+) where {FT}
+    return albedo_dry * (1 - surface_liquid_fraction) +
+           albedo_wet * surface_liquid_fraction
+end
+
+
+"""
+    update_albedo!(bc::AtmosDrivenFluxBC, p, soil_domain, model_parameters)
+
+Calculates and updates PAR and NIR albedo as a function of volumetric soil water content at
+the top of the soil. If the soil layers are very large, the water content of the top layer
```

Comment:
If the soil layers are larger than a specified parameter `alb...`, the water content of the top layer...